src/main/java/tds/exam/results/web/endpoints/TISCallbackController.java

```diff
@@ -0,0 +1,44 @@
+package tds.exam.results.web.endpoints;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.UUID;
+
+import tds.exam.results.services.MessagingService;
+import tds.exam.results.tis.TISState;
+
+@RestController
+public class TISCallbackController {
+    private final MessagingService messagingService;
+
+    @Autowired
+    public TISCallbackController(final MessagingService messagingService) {
+        this.messagingService = messagingService;
+    }
+
+    @RequestMapping(value = "/tis", method= RequestMethod.POST, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I assume you're pretty locked in by TIS expectations so feel free to ignore.  It would be nice if this had some type of identifier like `/tis/{examid}` and would do a PUT instead of POST.  I feel like you're updating the TIS "resource" with a status.